### DISCUSSION TITLE
Tailwindcss Not working with Nextjs after vercel Deployment

### DISCUSSION THREAD

**Discussion by mayureshsmitasuresh** [tailwindlabs/tailwindcss#6110] [Help]
Hello everyone
I am building a blog site using nextJs and tailwind CSS but facing one issue, after deploying to vercel tailwindcss is not working ..
Please refer screenshots below to understand

my package.json

![tailwind 2](https://user-images.githubusercontent.com/60806073/142037765-233c26bc-5435-4c37-8fb2-65151bc825df.JPG)

tailwindcss.config.js
![tailwind 1](https://user-images.githubusercontent.com/60806073/142037794-1c0dca09-1b79-4371-8cb5-748ae51a5d4a.JPG)
globals.css

tailwind 3
![tailwind 3](https://user-images.githubusercontent.com/60806073/142037846-be45f3a7-e4ea-43a4-9eff-a9b10b5db8b2.JPG)

I don't know why this is happening I tried a lot of solutions but still getting like this

![11](https://user-images.githubusercontent.com/60806073/142244195-4a71e7f7-981d-4325-8dd9-b749da569cb4.JPG)

locally it's like this

![222](https://user-images.githubusercontent.com/60806073/142244258-70a93ee9-964f-47c1-b8c3-5285e8aef459.JPG)
please give me a solution for that

**Comment by jparw3** [comment]


**Reply by AbdushukurRasulov** [reply]
@Jparw Did you face the same issue ?
I have the same issue, ring and some shadow class is not working after deploy

**Reply by jparw3** [reply]
> @Jparw Did you face the same issue ?
> 
> I have the same issue, ring and some shadow class is not working after deploy

I've managed to sort out the issues now after visiting tons of sites haha, just made sure that the content in tailwind config was 100% an accurate path... also making sure that tailwind was in dependencies and not dev dependencies

**Reply by AbdushukurRasulov** [reply]
Did as you said, but not worked

**Comment by arturs706** [comment]
Mine is not working, and not sure why. Any hints? I think I have issues with my own theme

**Reply by AbdushukurRasulov** [reply]
**just made sure that the content in tailwind config was 100% an accurate path... also making sure that tailwind was in dependencies and not dev dependencies**

**Reply by saiful-shaakil** [reply]
bro, I tried but it's not working also.
